backend-api/app/app.py

```diff
@@ -12,6 +13,12 @@
     allow_headers=["*"],
 )
 
+# Store smart contract state on the app instance
+minter = get_minter()
+token_contract = deploy_contract(minter)
+app.state.minter = minter
+app.state.token_contract = token_contract
```

Comment:
Cool! I didn't know the FastAPI instance had a state attribute that we could add data to.